### PR TITLE
#790 improve readme

### PR DESCRIPTION
**Pull request by jerzydziewierz**
As promised, an updated README that explains how to force lower memory usage.

### CODE REVIEW

**Comment by technovangelist** [comment]
Thanks for this contribution. Ideally the readme should apply to everyone, but this seems specific to Linux and maybe Windows with WSL. Since some parts are already in the Modelfile doc, can you update just whats needed to go into the FAQ? I think it would be a great addition there.

**Comment by technovangelist** [comment]
oh, and also remove the .gitignore file from the pr. thanks so much.

**Review by skydiver1980 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -117,6 +117,58 @@ ollama run mario
...
+2. Add the settings you want to change to the modelfile using your favorite editor
+
+For example, to attempt to load 20 layers into the GPU, and leave the rest of them to the CPU; and to use only 4 CPU threads, add the following lines to the modelfile:
+
+```yaml
+PARAMETER num_gpu 20
+PARAMETER num_thread 4
+```
+
+Note that if you use a non-dedicated GPU (e.g. a laptop GPU), you may need to leave some free VRAM for other applications to use,
```
Comment (jmorganca) [inline README.md:140]: This section could work great in the https://github.com/jmorganca/ollama/blob/main/docs/faq.md as Ollama should, over time, get better at scheduling these layers automatically.

**Comment by jmorganca** [comment]
Hi there, going to close this in favor of #1294 as it now allows setting parameters in `ollama run` directly. Thanks so much for the PR. RE the layer reference, this is quite interesting and would be a great addition to the FAQ, but is a little bit deep for the initial README.md

**Pull request closed without merging**